contrib/init/sysvinit-debian/docker

```diff
@@ -19,7 +19,7 @@ set -e
 
 export PATH=/sbin:/bin:/usr/sbin:/usr/bin:/usr/local/sbin:/usr/local/bin
 
-BASE=docker
+BASE=dockerd
 
 # modify these in /etc/default/$BASE (/etc/default/docker)
 DOCKER=/usr/bin/$BASE
```

Comment:
I think the same concern applies here, except with `/etc/default/docker` -- existing files would be broken by this change unless we rename the variable too, right?